src/core/js/views/notifyView.js

```diff
@@ -200,6 +200,8 @@ define([
         },
 
         closeNotify: function (event) {
+            if (!this.model.get('_isActive')) return;
```

Comment:
Picky I know but `_isOpen` feels like a better name...?

Also, could you add a comment:
```
//prevent from being invoked multiple times - see https://github.com/adaptlearning/adapt_framework/issues/1659
```